package.json

```diff
@@ -148,25 +137,19 @@
   "os": ["darwin", "linux", "win32"],
   "scripts": {
     "compile": "./build.sh",
+    "edition:lite": "git reset && git checkout package.json yarn.lock && rm -f .pro-mode",
```

Comment:
I think we don't want to `git reset` here as we'll lose pending changes we are trying to commit?

Reply:
Not loosing (it'd happen with `--hard` flag), but unstaging changes so that I can checkout files we need.

Reply:
God you